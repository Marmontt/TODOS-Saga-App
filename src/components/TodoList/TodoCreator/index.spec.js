import React from "react";
import { shallow } from "enzyme";

import { TodoCreator } from "./index";

describe("TodoCreator", () => {
  const props = {
    color: "red",
    text: "some-test-text",
    fulfilled: false,
    todoIndex: 0,
    setFulfilledState: jest.fn()
  };

  it("should render properly", () => {
    const wrapper = shallow(<TodoCreator {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
