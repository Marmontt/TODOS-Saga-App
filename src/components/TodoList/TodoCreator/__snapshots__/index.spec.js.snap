// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoCreator should render properly 1`] = `
<div
  className="todoCreator-container"
  style={
    Object {
      "background": "red",
    }
  }
>
  <div
    className="todoCreator-content"
  >
    <input
      checked={false}
      className="todoCreator-checkbox"
      onChange={[Function]}
      type="checkbox"
    />
    <div
      className="todoCreator-text"
    >
      some-test-text
    </div>
  </div>
</div>
`;
